docs/general/kusama/kusama-statemine.md

```diff
@@ -5,87 +5,80 @@ sidebar_label: Statemine
 description: Statemine and its features
 slug: ../../kusama-statemine
 ---
+
 import RPC from "./../../../components/RPC-Connection"
 
-Statemine is a generic assets parachain which provides functionality for deploying and transferring assets — both Fungible and Non-Fungible Tokens (NFTs). It is a common good parachain on Kusama (not to be confused with [Statemint](../../learn/learn-common-good-chains.md#statemint), which is a parachain on Polkadot). The native token of Statemine is KSM. Balance transfers and the Existential Deposit (ED) of KSM on Statemine are about 1/10th of the values on the Relay chain. For example, the Existential Deposit of a Statemine account is <RPC network="statemine" path="consts.balances.existentialDeposit" defaultValue={3333333} filter="humanReadable"/>, 
-when compared to <RPC network="kusama" path="consts.balances.existentialDeposit" defaultValue={33333333} filter="humanReadable"/>, on Kusama.
-Apart from the core protocol token KSM, the assets held on Statemine can be broadly categorized as 
+Statemine is a generic assets parachain which provides functionality for deploying and transferring
+assets — both Fungible and Non-Fungible Tokens (NFTs). It is a common good parachain on Kusama (not
+to be confused with [Statemint](../../learn/learn-common-good-chains.md#statemint), which is a
+parachain on Polkadot). The native token of Statemine is KSM. Balance transfers and the Existential
+Deposit (ED) of KSM on Statemine are about 1/10th of the values on the Relay chain. For example, the
+Existential Deposit of a Statemine account is
+<RPC network="statemine" path="consts.balances.existentialDeposit" defaultValue={3333333} filter="humanReadable"/>,
+when compared to
+<RPC network="kusama" path="consts.balances.existentialDeposit" defaultValue={33333333} filter="humanReadable"/>,
+on Kusama. Apart from the core protocol token KSM, the assets held on Statemine can be broadly
```

Comment:
@filippoweb3 
I was curious how prettify would handle the RPC components.  This is going to cause an issue with formatting:

![image](https://user-images.githubusercontent.com/13341935/186683441-7c735bac-6063-4ae6-96a6-eb16ee0a73d2.png)

The behavior is pretty strange, both instance exhibit slightly different formatting (newline vs no space).  I need to spend a few minutes looking into it.  We can probably make a change on the RPC component to always force inline behavior, unless maybe a double newline is used.

Reply:
Good catch! Let me know if it is possible to fix it!

Reply:
Looks like the markdown parser expects the newline to start with a character, if it doesn't it assumes end of paragraph and adds a bottom margin to that paragraph:

![image](https://user-images.githubusercontent.com/13341935/186699364-51bde004-6581-4860-9484-0cbefc1df143.png)

If you put a character infront of `<RPC` it formats correctly.  I wonder if the same behaviour happens when using the `{{ polkadot: :polkadot }}` syntax.

Reply:
Since the formatting comes from the parent component, not the RPC component, I am hesitant to mess with any higher-level CSS.  A small change that works here could incorrectly format other content on the site.

One not so great option is to ignore the prettify command on blocks that contain RPC components such as:

![image](https://user-images.githubusercontent.com/13341935/186712564-0242ff37-b3e7-4cf6-b615-ad893bc16c61.png)

It works but redundant and not ideal.  For now I am going to manually fix these couple occurrences and file another issue around finding a better long-term solution.  I am hoping there is a way to add a rule to prettier to ignore lines that contain HTML elements so we don't have to manually mark each occurrence.